src/display-buffer.coffee

```diff
@@ -1314,6 +1154,25 @@ class DisplayBuffer extends Model
       # this one. Only emit when the marker still exists.
       @emitter.emit 'did-create-marker', marker
 
+  # TODO: serialize state in TextEditorElement, rather than saving scroll
+  # positions here.
```

Comment:
Imagine if your font is a different size next time you load the editor... don't you actually want the same logical location rather than the same pixel location to be in view? That's what makes me wonder about expressing the scroll top as state in terms of rows... but maybe it's something that is only assigned by the view and read one time when it is attached to the DOM.